<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>delHis</class>
 <widget class="QDialog" name="delHis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>216</width>
    <height>42</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>지우기</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(240, 245, 221);</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 245, 158);</string>
   </property>
   <property name="text">
    <string>지우기</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
